test/spec/userSync_spec.js

```diff
@@ -250,6 +265,7 @@ describe('user sync', function () {
 
   it('should throw a warning and default to basic resgistration rules when filterSettings config is invalid', function () {
     // invalid config - passed invalid filter option
+    debugger; // eslint-disable-line
```

Comment:
Remove this breakpoint

Reply:
Thanks for checking that, I've removed it now.